src/Majima/Majima.java

```diff
@@ -0,0 +1,46 @@
+package Majima;
+
+import Majima.command.Command;
+import Majima.ui.Ui;
```

Comment:
might want to consider sorting imports according to lexical order

Reply:
Fixed by right clicking the project directory in IntelliJ and optimizing imports from the dropdown menu/CTRL + ALT + O (windows)